src/main/kotlin/no/nav/familie/ef/sak/api/ApiExceptionHandler.kt

```diff
@@ -38,6 +38,13 @@ class ApiExceptionHandler {
         return ResponseEntity.status(feil.httpStatus).body(Ressurs.failure(frontendFeilmelding = feil.frontendFeilmelding))
     }
 
+    @ExceptionHandler(ManglerTilgang::class)
+    fun handleThrowable(manglerTilgang: ManglerTilgang): ResponseEntity<Ressurs<Nothing>> {
+        secureLogger.error("En håndtert tilgangsfeil har oppstått - ${manglerTilgang.melding}", manglerTilgang)
+        logger.info("En håndtert tilgangsfeil har oppstått")
+        return ResponseEntity.status(HttpStatus.OK).body(Ressurs.ikkeTilgang(melding = manglerTilgang.melding))
```

Comment:
Synes det er litt merkelig å si ok når man mangler tilgang. Burde ikke det vært unauthorized?

Reply:
Vi sier "ok" på feil også, men pakker feil/ikke tilgang inn i Ressurs istedenfor.

Reply:
Det synes jeg blir helt feil. Jeg syens jo i utgangspunktet at Ressurs var en dårlig idé, men å pakke feil inn i ok i stedet er jo en elendig idé. Da må man jo belage seg på at ok kan bety feil å lage masse egen kode for å håndtere feil i stedet for å bruke bibliotekene slik de er tiltenkt.

Reply:
Jeg tror egentlig frontend kun forholder seg til ressurs-statusen. Hvis man sender feilkode i tillegg, kan det bli tvetydig - feks 500 kombinert med en ressurs.ok 🤷‍♀️  Her kan vel de som har jobbet med frontend kanskje si noe om hvilket mønster de synes er best.

For øvrig @bent-lorentzen : Hvis du mener Ressurs er en dårlig idé, er du hjertelig velkommen til å ta det opp i plenum på feks teknisk review med tilhørende argumenter. Vi har så og si udelt positiv erfaring med å bruke Ressurs i ba-sak.

Reply:
Etter litt nærmere sjekk ser jeg at frontend fint håndterer 4XX/5XX med Ressurs, så vi kan godt endre dette.

Reply:
Den blir FORBIDDEN - ettersom brukeren er authorized, men ikke har tilgang til akkurat denne delen.